Framework/CurveFitting/src/IMWDomainCreator.cpp

```diff
@@ -186,6 +186,11 @@ std::pair<size_t, size_t> IMWDomainCreator::getXInterval() const {
     }
   }
 
+  if (to - from == 0) {
```

Comment:
I was looking where from and to came from - could the if statement above be simplified so that this is a bit clearer?

Could there be a comment explaining `to - from == 0`?